plugins/inputs/ravendb/ravendb.go

```diff
@@ -0,0 +1,680 @@
+		"disk_system_store_used_data_file_size_in_mb":       serverResponse.Disk.SystemStoreUsedDataFileSizeInMb,
+		"disk_system_store_total_data_file_size_in_mb":      serverResponse.Disk.SystemStoreTotalDataFileSizeInMb,
+		"disk_total_free_space_in_mb":                       serverResponse.Disk.TotalFreeSpaceInMb,
+		"disk_remaining_storage_space_percentage":           serverResponse.Disk.RemainingStorageSpacePercentage,
+		"license_type":                                      serverResponse.License.Type,
+		"license_utilized_cpu_cores":                        serverResponse.License.UtilizedCpuCores,
+		"license_max_cores":                                 serverResponse.License.MaxCores,
+		"network_tcp_active_connections":                    serverResponse.Network.TcpActiveConnections,
+		"network_concurrent_requests_count":                 serverResponse.Network.ConcurrentRequestsCount,
+		"network_total_requests":                            serverResponse.Network.TotalRequests,
+		"network_requests_per_sec":                          serverResponse.Network.RequestsPerSec,
+		"cluster_node_state":                                mapNodeState(serverResponse.Cluster.NodeState),
+		"cluster_current_term":                              serverResponse.Cluster.CurrentTerm,
+		"cluster_index":                                     serverResponse.Cluster.Index,
+		"databases_total_count":                             serverResponse.Databases.TotalCount,
+		"databases_loaded_count":                            serverResponse.Databases.LoadedCount,
+	}
+
+	if serverResponse.Config.TcpServerUrls != nil {
+		fields["config_tcp_server_urls"] = strings.Join(serverResponse.Config.TcpServerUrls, ";")
+	}
+
+	if serverResponse.Config.PublicTcpServerUrls != nil {
+		fields["config_public_tcp_server_urls"] = strings.Join(serverResponse.Config.PublicTcpServerUrls, ";")
+	}
+
+	if serverResponse.Cpu.MachineIoWait != nil {
+		fields["cpu_machine_io_wait"] = serverResponse.Cpu.MachineIoWait
+	}
+
+	if serverResponse.License.ExpirationLeftInSec != nil {
+		fields["license_expiration_left_in_sec"] = serverResponse.License.ExpirationLeftInSec
+	}
+
+	if serverResponse.Network.LastRequestTimeInSec != nil {
+		fields["network_last_request_time_in_sec"] = serverResponse.Network.LastRequestTimeInSec
+	}
+
+	if serverResponse.Network.LastAuthorizedNonClusterAdminRequestTimeInSec != nil {
+		fields["network_last_authorized_non_cluster_admin_request_time_in_sec"] = serverResponse.Network.LastAuthorizedNonClusterAdminRequestTimeInSec
+	}
+
+	if serverResponse.Certificate.WellKnownAdminCertificates != nil {
+		fields["certificate_well_known_admin_certificates"] = strings.Join(serverResponse.Certificate.WellKnownAdminCertificates, ";")
+	}
+
+	if serverResponse.Certificate.ServerCertificateExpirationLeftInSec != nil {
+		fields["certificate_server_certificate_expiration_left_in_sec"] = serverResponse.Certificate.ServerCertificateExpirationLeftInSec
+	}
```

Comment:
You might just add the fields as `acc` will ignore fields with a `nil` value.

Reply:
I leave items which does strings.join inside if.

Reply:
Thx, please mark as resolved.